src/app/crud/base.py

```diff
@@ -50,7 +50,7 @@ async def get(self, entry_id: int, strict: bool = False) -> Union[ModelType, Non
 
     async def get_by(self, field_name: str, val: Union[str, int], strict: bool = False) -> Union[ModelType, None]:
         statement: Select = select(self.model).where(getattr(self.model, field_name) == val)
-        results = await self.session.execute(statement=statement)
+        results = await self.session.exec(statement=statement)  # type: ignore
```

Comment:
Also let's no ignore all typing issues and be specific if we ignore any
Not sure why it was worth changing this line, since we're now ignore mypy on this line? Perhaps safer to address this in a more specific PR (if the only CI problem is with mypy temporarily, I think it's quite acceptable)

Reply:
oh yes sorry for that I forgot th precise the mypy error to ignore